cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -96,7 +102,10 @@ <h1>
                 </div>
             </div>
             <div class="container" id="selected-experiments">
-                <div class="text-xl font-bold text-slate-500">Selected Experiments</div>
+                <div class="flex items-center">
```

Comment:
I don't think these changes are reflected in the screenshot.

Reply:
Added screentshot